bundles/org.openhab.binding.lghombot/src/main/java/org/openhab/binding/lghombot/internal/LGHomBotHandler.java

```diff
@@ -0,0 +1,722 @@
+                        break;
+                    case "WEDNESDAY":
+                        wednesday = state;
+                        break;
+                    case "THURSDAY":
+                        thursday = state;
+                        break;
+                    case "FRIDAY":
+                        friday = state;
+                        break;
+                    case "SATURDAY":
+                        saturday = state;
+                        break;
+                    case "SUNDAY":
+                        sunday = state;
+                        break;
+                    default:
+                        break;
+                }
+
+            }
+            if (!currentMonday.equals(monday)) {
+                currentMonday = monday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_MONDAY);
+                updateState(channel, StringType.valueOf(monday));
+            }
+            if (!currentTuesday.equals(tuesday)) {
+                currentTuesday = tuesday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_TUESDAY);
+                updateState(channel, StringType.valueOf(tuesday));
+            }
+            if (!currentWednesday.equals(wednesday)) {
+                currentWednesday = wednesday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_WEDNESDAY);
+                updateState(channel, StringType.valueOf(wednesday));
+            }
+            if (!currentThursday.equals(thursday)) {
+                currentThursday = thursday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_THURSDAY);
+                updateState(channel, StringType.valueOf(thursday));
+            }
+            if (!currentFriday.equals(friday)) {
+                currentFriday = friday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_FRIDAY);
+                updateState(channel, StringType.valueOf(friday));
+            }
+            if (!currentSaturday.equals(saturday)) {
+                currentSaturday = saturday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_SATURDAY);
+                updateState(channel, StringType.valueOf(saturday));
+            }
+            if (!currentSunday.equals(sunday)) {
+                currentSunday = sunday;
+                channel = new ChannelUID(getThing().getUID(), CHANNEL_SUNDAY);
+                updateState(channel, StringType.valueOf(sunday));
+            }
+        }
+    }
+
+    private void parseImage() {
+        if (!isLinked(CHANNEL_CAMERA)) {
+            return;
+        }
+        final int width = 320;
+        final int height = 240;
+        final int size = width * height;
+
+        BufferedImage image = new BufferedImage(width, height, BufferedImage.TYPE_INT_RGB);
+        String url = buildHttpAddress("/images/snapshot.yuv");
+        byte[] yuvData = HttpUtil.downloadData(url, null, false, size * 2).getBytes();
```

Comment:
Does download image not work here? Consider putting this in a dedicted class

Reply:
It's a raw yuv-image at 320x240 pixels, last time (a year ago) I looked I couldn't find anything regarding yuv conversion or loading of raw images, has that changed?
I will put it in a separate class for now.

Reply:
Indeed still no support for yuv I scanned to quick and thought it was some more standard format